packages/runtime/src/errors/errors-list.ts

```diff
@@ -316,55 +329,63 @@ const asaErrors = {
     Please check your ASA and config files`
   },
   ASSET_NOT_FOUND: {
-    number: 902,
+    number: 1502,
     message: `Asset with Index %assetId% not found`,
     title: "Asset Not Found",
     description: "Asset Not Found"
   },
-  ACCOUNT_ASSET_FROZEN: {
-    number: 903,
-    message: `Asset index %assetId% frozen for account %address%`,
-    title: "Asset Frozen",
-    description: "Asset Frozen for account"
+  MAX_LIMIT_ASSETS: {
+    number: 1503,
+    message: "Error while creating asset %name% for %address%. Maximum created assets for an account is %max%",
+    title: 'Asset Creation Error',
+    description: `Asset Creation Error`
   },
   MANAGER_ERROR: {
-    number: 904,
+    number: 1504,
     message: `Only Manager account %address% can modify asset`,
     title: "Manager Error",
     description: "Manager Error"
   },
   FREEZE_ERROR: {
-    number: 905,
+    number: 1505,
     message: `Only Freeze account %address% can freeze asset`,
     title: "Freeze Error",
     description: "Freeze Error"
   },
   CLAWBACK_ERROR: {
-    number: 906,
+    number: 1506,
     message: `Only Clawback account %address% can revoke asset`,
     title: "Clawback Error",
     description: "Clawback Error"
   },
   BLANK_ADDRESS_ERROR: {
-    number: 907,
+    number: 1507,
     message: `Cannot reset a blank address`,
     title: "Blank Address Error",
     description: "Blank Address Error"
   },
   ASSET_TOTAL_ERROR: {
-    number: 908,
+    number: 1508,
     message: "All of the created assets should be in creator's account",
     title: "Asset Total Error",
     description: "Asset Total Error"
   }
 };
 
-export const ERRORS: {
-  [category in keyof typeof ERROR_RANGES]: {
+export const TEAL_ERRORS: {
+  [category in keyof typeof TEAL_ERROR_RANGES]: {
+    [errorName: string]: ErrorDescriptor
+  };
+} = {
+  TEAL: tealErrors
+};
+
+export const RUNTIME_ERRORS: {
+  [category in keyof typeof RUNTIME_ERROR_RANGES]: {
     [errorName: string]: ErrorDescriptor
   };
 } = {
-  TEAL: tealErrors,
+  GENERAL: runtimeGeneralErrors,
   TRANSACTION: transactionErrors,
-  ASA: asaErrors
+  ASA: runtimeAsaErrors
```

Comment:
Same as above. Is there a reason to keep TEAL errors separate?

Reply:
Just that in opcode list we are using teal opcode errors, and in runtime and account we throw runtime errors